cea/plots/__init__.py

```diff
@@ -99,7 +113,11 @@ def __init__(self, config, dashboard_dict, cache):
         try:
             self.description = dashboard_dict['description']
         except KeyError:
-            self.description = None
+            self.description = ''
+        try:
+            self.layout = dashboard_dict['layout']
+        except KeyError:
+            self.layout = 'default'
```

Comment:
again. here we're introducing a third value for `self.layout`... which i guess maps to rows?